quickstart/src/userfriendlyerror.js

```diff
@@ -2,24 +2,27 @@
 
 const USER_FRIENDLY_ERRORS = {
   NotAllowedError: () => {
-    return 'The user did not give permission to access your media.';
+    return '<b>Cause: </b><br>The user did not give permission to access to your input device by clicking the "deny" button on the permission dialog, or dismissing the permission dialog, or by going to the browser settings.<br>'
+    +'<br><b>Solution: </b><br> Please allow the app to access the input device on the browser settings and reload the app.';
   },
   NotFoundError: () => {
-    return 'None of the available media types satisfied the given constraints.';
+    return '<b>Cause: </b><br>The user has disabled the input device for the browser in the system settings or the user\'s machine does not have such input device connected to it.<br>'
+    +'<br><b>Solution</b><br>The user should enable the input device for the browser in the system settings, or have atleast one input device connected.';
   },
   NotReadableError: () => {
-    return 'Could not access your media due to a hardware error.';
+    return '<b>Cause: </b><br>The browser could not start media capture with the input device.<br>'
+    +'<br><b>Solution: </b><br>Please close other apps and windows that have reserved the input device and reload the app or restart the browser.';
```

Comment:
### Cause
The browser could not start media capture with the input device even after the user gave permission, probably because another app or tab has reserved the input device.

### Solution
The user should close all other apps and tabs that have reserved the input device and reload your app, or worst case, restart the browser.